### PR TITLE
11...3-157 npc scripts

### PR DESCRIPTION
**Pull request by gwagwaflamingo**
Quest state dependant scripts commented out for the future.

### CODE REVIEW

File: Npc/11000040.py [hunk 1]
```diff
@@ -5,8 +5,10 @@
 
 class Main(Script):
     def first(self) -> int:
-        # TODO: Job 1
-        return random.choice([10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 110, 120])
+        x = self.job()//10
```
Comment (kOchirasu) [inline Npc/11000040.py]: naming: `job_code` instead of `x`

File: Npc/npc_api.py [hunk 2]
```diff
@@ -41,6 +41,15 @@ def reward_item(self, rewards) -> bool:
...
         return self.ctx.HasItem(item_id, rarity)
-    
+
     def has_mesos(self, amount: int) -> bool:
         return self.ctx.HasMesos(amount)
+
+    def level(self) -> int:
+        return self.ctx.Level()
+
+    def job(self) -> int:
```
Comment (kOchirasu) [inline Npc/npc_api.py:51]: maybe add another function:
```py
def job_code(self) -> int:
    return self.ctx.Job()//10
```

Comment (gwagwaflamingo) [reply Npc/npc_api.py:51]: Should I just do something like this in NpcScriptContext instead? I don't think we'll ever need the jobcode non-divided by 10
```
public int Job() {
    return ((int)Session.Player.Value.Character.Job) / 10;
}

Comment (kOchirasu) [reply Npc/npc_api.py:51]: I would keep it how it currently is because Job has strictly more information than JobCode.

Comment (gwagwaflamingo) [reply Npc/npc_api.py:51]: Got it, made the changes 😊

File: Npc/11000075.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 class Main(Script):
     def first(self) -> int:
-        return random.choice([10, 20, 30, 40])
+        return 20
```
Comment (kOchirasu) [inline Npc/11000075.py:9]: what about 10, 30, 40? Are they completely unused? We should keep some documentation to not lose track of these starting states.

File: Npc/11000096.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 class Main(Script):
     def first(self) -> int:
-        return random.choice([30, 40, 50])
+        return 40
```
Comment (kOchirasu) [inline Npc/11000096.py:9]: same as above

Comment (gwagwaflamingo) [reply Npc/11000096.py:9]: Added context for not including them, will continue doing so in the future

**Pull request merged**